Jenkinsfile

```diff
@@ -500,20 +500,20 @@ pipeline {
                         }
                     }
                 }
-                stage('Build on CentOS 7') {
+                stage('Build on EL 8') {
                     when {
                         beforeAgent true
                         expression { ! skipStage() }
                     }
                     agent {
                         dockerfile {
-                            filename 'utils/docker/Dockerfile.centos.7'
+                            filename 'utils/docker/Dockerfile.el.8'
```

Comment:
It would be interesting to be able to get the Dockerfile name extension from the stage name via `parseStageInfo()` somehow so that we could get to a point of only having to change the distro name in the stage name to make these kinds of changes.

Reply:
That should be possible for a future PR.  Just need to get a single string from the map returned, once the map  has that information.

Reply:
Indeed.  Was not at all expecting that here.  Just planting the seed for a future enhancement.